### PR TITLE
Feature/grammar construction from keywords

### PR DESCRIPTION
**Pull request by byung-o**

### CODE REVIEW

File: src/daidepp/keywords/press_keywords.py [hunk 1]
```diff
@@ -210,6 +61,11 @@ class ALYVSS(_DAIDEObject):
     aly_powers: List[Power]
     vss_powers: List[Power]
 
+    def __init__(self, aly_power, vss_powers):
```
Comment (mjspeck) [inline src/daidepp/keywords/press_keywords.py:64]: Why is this necessary? Also, shouldn't `aly_power` be plural? (`aly_powers`)

File: src/daidepp/keywords.py [hunk 2]
```diff
@@ -625,9 +470,6 @@ def __str__(self):
     REJ,
     BWX,
     HUH,
-    FCT,
```
Comment (mjspeck) [inline src/daidepp/keywords.py:628]: Why are these removed?

File: src/daidepp/keywords/press_keywords.py (hunk 1 above)
Comment (byung-o) [reply src/daidepp/keywords/press_keywords.py:64]: not necessary. Idk why this is in there

File: src/daidepp/keywords.py (hunk 2 above)
Comment (byung-o) [reply src/daidepp/keywords.py:628]: These were in the list twice

File: tests/test_daide_visitor.py [hunk 3]
```diff
@@ -18,6 +18,7 @@ def test_basic_visitor(sample_daide_messages: List[str]):
 
 
 @pytest.mark.parametrize(
+    # fmt: off
```
[the remark this answers is not in the document]
Comment (byung-o) [reply tests/test_daide_visitor.py:21]: When I ran `black` on the tests, these the parameters for each test would multiple lines, which I though looked bad. So this makes `black' ignore this portion.
